nginx.conf

```diff
@@ -1,6 +1,15 @@
 server {
     listen 80;
-    server_name privacybadger.com www.privacybadger.com privacybadger.org www.privacybadger.org localhost;
+    server_name privacybadger.com www.privacybadger.com;
+
+    location / {
+        return 301 https://privacybadger.org$request_uri;
+    }
+}
+
+server {
+    listen 80;
+    server_name privacybadger.org www.privacybadger.org localhost;
```

Comment:
Should we also redirect the www subdomain?

Reply:
I think so. Having content duplicated on multiple domains like this can cause problems with search rankings.

Reply:
to clarify, you're saying www.privacybadger.org should redirect to privacybadger.org?

Reply:
Yes. Isn't it the same issue? We want one canonical domain only.

Reply:
sounds good, updated the pr